services/ui/ws/window_tree.cc

```diff
@@ -910,7 +910,7 @@ void WindowTree::ProcessWindowBoundsChanged(
   }
 
   client()->OnWindowBoundsChanged(ClientWindowIdToTransportId(client_window_id),
-                                  old_bounds, new_bounds, local_surface_id);
+                                  old_bounds, newest_bounds, local_surface_id);
```

Comment:
Excuse me, but how is it related to the commit you've mentioned in the commit message?

Reply:
I think you are right. this should be related to "fixup! Handle OnBoundsChange in external mode.".

Reply:
I'll fix it during rebasing if you don't mind.